unstructured/ingest/main.py

```diff
@@ -781,14 +790,15 @@ def main(
         logger.error("No connector-specific option was specified!")
         sys.exit(1)
 
-    process_document_with_partition_strategy = partial(
+    process_document_with_partition_args = partial(
         process_document,
-        strategy=partition_strategy,
+        partition_strategy=partition_strategy,
```

Comment:
I don't think we want to update the name of this arg from `strategy`->`partition_strategy`, right? since this will eventually trickle via kwargs to [here](https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/partition/auto.py#L44)

Reply:
This was a typo on my part. Thank you for catching it!